src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -58,6 +58,16 @@ export const monoPlatform: Platform = {
     return new Uint8Array(Module.HEAPU8.buffer, dataPtr + 4, length);
   },
 
+  toDotNetString: (string: String): System_String => {
+    const binding = window['BINDING'];
+    return binding.js_string_to_mono_string(string);
+  },
```

Comment:
While you're adding this kind of thing, would you mind checking if there's a similar (and better) implementation we could put into `toJavaScriptString` in this file?

The current implementation contains a weird comment we inherited from the Mono sources:

```
    // FIXME this is wastefull, we could remove the temp malloc by going the UTF16 route
    // FIXME this is unsafe, cuz raw objects could be GC'd.
```

If there's a "non-wastefull" version of this logic in `BINDING`, we should consider using it.

Also, rather than using `window['BINDING']` in a loosely-typed way, would you consider declaring `BINDING` inside `MonoTypes.d.ts` so those functions can just be called using normal strongly-typed syntax? Doesn't need to list *all* of the `BINDING` APIs; just the ones we're using. We can add to it over time.